tutorials/WithTypescript.md

```diff
@@ -0,0 +1,101 @@
+# [Basic usage](https://github.com/jurijtokarski/react-source-of-truth/blob/master/tutorials/Usage.md) with TypeScript
```

Comment:
replace this link with https://dzek69.github.io/react-source-of-truth/tutorial-Usage.html ;)